neuro_extras/main.py

```diff
@@ -217,6 +222,54 @@ def image_copy(source: str, destination: str) -> None:
     run_async(_copy_image(source, destination))
 
 
+@storage.command("copy")
```

Comment:
let's have `neuro-extras cp` similar to `neuro cp` as it just extends `neuro cp`'s functionality?

Reply:
done